scapy/contrib/modbus.uts

```diff
@@ -13,34 +13,43 @@ str(ModbusADURequest() / '\x00\x01\x02') == '\x00\x00\x00\x00\x00\x04\xff\x00\x0
 
 = MBAP Guess Payload ModbusPDU01ReadCoilsRequest (simple case)
 p = ModbusADURequest('\x00\x00\x00\x00\x00\x06\xff\x01\x00\x00\x00\x01')
-assert(isinstance(p.payload, ModbusPDU01ReadCoilsRequest))
+p.payload.__class__.__name__ == 'ModbusPDU01ReadCoilsRequest'
```

Comment:
A bad merge from an old scapy and mine. I didn't notice it was this kind of modifications.